src/bp/core/services/cms.ts

```diff
@@ -582,7 +582,10 @@ export class CMSService implements IDisposeOnExit {
 
     if (originalProps) {
       return originalProps.reduce((result, key) => {
-        result[key] = formData[`${key}$${lang}`] || (defaultLang && formData[`${key}$${defaultLang}`])
+        result[key] =
```

Comment:
I get what you do and why you do this but you'll agree with me that this is quite ugly ...